bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/conf/TelldusLocalConfiguration.java

```diff
@@ -12,14 +12,17 @@
  */
 package org.openhab.binding.tellstick.internal.conf;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
 /**
  * Configuration class for Tellstick bridge used to connect to the
  * Telldus local API.
  *
  * @author Jan Gustafsson - Initial contribution
  */
+@NonNullByDefault
 public class TelldusLocalConfiguration {
-    public String ipAddress;
-    public String accessToken;
+    public String ipAddress = "";
+    public String accessToken = "";
     public long refreshInterval;
```

Comment:
Perhaps the default:
https://github.com/openhab/openhab-addons/blob/d14749bb64f3cd4931f5c1aabb246083602fb33e/bundles/org.openhab.binding.tellstick/src/main/resources/OH-INF/thing/bridge.xml#L76

should be added here as well?
```suggestion
    public long refreshInterval = 60000;
```

Reply:
Fixed